### ISSUE TITLE
Can I configure which matchers run, by specific rule id or impact?

### ISSUE THREAD

**Issue by davidmason** [dequelabs/axe-matchers#81]
Great library. We have a long way to go fixing up accessibility so I want to tackle it guideline-by-guideline and make sure our specs will fail if anyone makes changes that fail one of the guidelines I've already dealt with. To do that I'll need to turn on specific rules one by one.

I can't see documentation for how I can configure axe-matchers to include/exclude specific rules, or set an impact level to test for (Critical/Serious/Minor).

- Is it possible to configure like that?
- If so, is it documented somewhere? I couldn't find anything linked from the readme.

**Comment by davidmason** [comment]
It looks like this might answer my questions: https://github.com/dequelabs/axe-matchers/blob/develop/docs/RSpec.md and it is linked from the readme, so I'll close this.
